lib/puppet/application/cert.rb

```diff
@@ -74,6 +74,10 @@ def subcommand=(name)
     options[:allow_dns_alt_names] = value
   end
 
+  option("--[no-]allow-allow-authorization-extensions") do |value|
+    options[:allow_dns_alt_names] = value
```

Comment:
`allow_dns_alt_names` seems like the wrong symbol to use here. Did we want `allow_authorization_extensions`?